### ISSUE TITLE
umami.is DNS down

### ISSUE THREAD

**Issue by itsmingjie** [mikecao/umami#105]
** fully aware that this is not an issue with the repository, but this is probably the only way I can bring it up.

I have no idea what happened, but 2 hours ago I was looking at the docs normally on umami.is, but suddenly the docs are gone and the DNS records disappeared. Can someone please look into this? 

Umami is awesome, btw!

![image](https://user-images.githubusercontent.com/13333578/92282074-ecb04180-eeca-11ea-9070-5b27d86ad894.png)
![image](https://user-images.githubusercontent.com/13333578/92282085-f8036d00-eeca-11ea-96c2-c2898bf330c3.png)

**Comment by junaid33** [comment]
Noticed the same thing. I would use something like Uptimebot so you know if the website goes down.

**Comment by itsmingjie** [comment]
Yeah interestingly it's not even the website itself -- the domain literally stopped resolving
